test/i-frame.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { browser } from 'protractor';
+import { IFramePage } from '../src/page';
+
+describe('Entering to the website', () => {
+  beforeAll(async () => {
+    await browser.get('https://demoqa.com/frames');
+  });
+  describe('Changing the iframe1 heigh', async () => {
+    const iframePage: IFramePage = new IFramePage();
```

Comment:
create a variable to represent the height, then use that variable to change the height and to make the assertion